das/aggregator.go

```diff
@@ -207,8 +212,21 @@ func (a *Aggregator) Store(ctx context.Context, message []byte, timeout uint64,
 	expectedHash := dastree.Hash(message)
 	for _, d := range a.services {
 		go func(ctx context.Context, d ServiceDetails) {
-			cert, err := d.service.Store(ctx, message, timeout, sig)
+			storeCtx, cancel := context.WithTimeout(ctx, a.requestTimeout)
+			defer cancel()
+			incFailureMetric := func() {
+				metrics.GetOrRegisterGauge("arb/das/rpc/aggregator/store/"+d.metricName+"/failure", nil).Inc(1)
```

Comment:
Would probably make sense to have a var for `"arb/das/rpc/aggregator/store/"+d.metricName` since it's repeated a lot. Maybe also a general const for `arb/das/rpc/aggregator/store/`

Reply:
Done